src/test/java/org/jenkinsci/plugins/workflow/multibranch/JobPropertyStepTest.java

```diff
@@ -296,6 +308,64 @@
         assertEquals("[null, false, null]", MockTrigger.startsAndStops.toString());
     }
 
+    @Test public void scmAndEmptyTriggersProperty() throws Exception {
+        WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "p");
+        // Verify the base case behavior.
+        p.setDefinition(new CpsFlowDefinition("echo 'foo'"));
```

Comment:
:ant: always pass an explicit `sandbox` argument, typically `true` for tests unless you specifically need to test the opposite,